internal/statesync/reactor.go

```diff
@@ -824,3 +840,28 @@ func (r *Reactor) fetchLightBlock(height uint64) (*types.LightBlock, error) {
 	}, nil
 
 }
+
+func (r *Reactor) GetLogger() log.Logger {
```

Comment:
Why do we need this?

Reply:
it's for the TestNodeStartStateSync, the mock instance is not able to retrieve Logger directly.